nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -42,11 +54,31 @@ void TfOpNode::setLayerNode(const LayerNode &layer) {
     }
     return v;
   };
+  /**
+   * Q1) Why convert from NCHW to NHWC?
+   * A1) the tflite uses NHWC format; nntrainer uses NCHW
+   *
+   * Q2) Why are only output tensors reshaped?
+   * A2) the tflite needs only one tensor between nodes. Therefore,
+   *basically, outputs are used for tflite tensors
+   **/
+  auto create_variables_with_NCHW_to_NHWC = [](auto tensor_getter,
+                                               unsigned size) {
+    Variables v;
+    v.reserve(size);
+    for (unsigned i = 0; i < size; ++i) {
+      Tensor *tensor = const_cast<Tensor *>(tensor_getter(i));
+      tensor->reshape(TensorDim{tensor->batch(), tensor->height(),
+                                tensor->width(), tensor->channel()});
+      v.push_back(tensor);
+    }
+    return v;
+  };
 
   inputs = create_variables(
```

Comment:
Because we are using only output tensor to export, do we need inputs? and that's why create_variables_with_NCHW_to_NHWC isn't called?